sam_bot_description/src/description/sam_bot_description.urdf

```diff
@@ -350,10 +341,13 @@
   </joint>
 
   <gazebo reference="camera_link">
-    <sensor name="depth_camera" type="depth">
+    <sensor name="depth_camera" type="depth_camera">
```

Comment:
Ditto with lidar :-)

Reply:
Will be adding this to the SDF file also.

Reply:
Should be good to go with [57db422](https://github.com/ros-navigation/navigation2_tutorials/pull/102/commits/57db4224d143056d692990a95f7cc3ad7a7cb45b).